### PR TITLE
[46]Movie/GIF planimation

### PR DESCRIPTION
**Pull request by Ojeechan**
made this PR to avoid Travis problem.

### CODE REVIEW

File: Assets/Visualiser/Scripts/ScenesCoordinator.cs [hunk 1]
```diff
@@ -81,10 +81,9 @@ public void planimationPlugin()
         byte[] formSections = UnityWebRequest.SerializeFormSections(formData, boundary);
         /* upload test for drag and drop function */
 
-        //UnityWebRequest www = UnityWebRequest.Post("http://127.0.0.1:8000/upload/pddl", formData);
-        UnityWebRequest www = UnityWebRequest.Post("/upload/pddl", formData);
-
-		
+        UnityWebRequest www = UnityWebRequest.Post("http://127.0.0.1:8000/upload/pddl", formData);
```
Comment (nirlipo) [inline Assets/Visualiser/Scripts/ScenesCoordinator.cs:84]: It should not point to localhost. Need to use the commented line pointing to the relative path

Comment (Ojeechan) [reply Assets/Visualiser/Scripts/ScenesCoordinator.cs]: rewrited the URL to "/upload/pddl"

Comment (nirlipo) [reply Assets/Visualiser/Scripts/ScenesCoordinator.cs]: Thanks!

**Comment by nirlipo** [comment]
@Ojeechan you can now create the PR from the branch to devel. Thanks!

**Pull request merged**